testing/middleware/builder/builder.go

```diff
@@ -581,6 +597,92 @@ func (p *Builder) handleHeaderRequestDeneb(w http.ResponseWriter) {
 	w.WriteHeader(http.StatusOK)
 }
 
+//func (p *Builder) handleHeaderRequestElectra(w http.ResponseWriter) {
```

Comment:
You don't handle execution requests here

Reply:
I added it